editor/animation_track_editor.cpp

```diff
@@ -2725,7 +2725,7 @@ void AnimationTrackEdit::_gui_input(const Ref<InputEvent> &p_event) {
 			path = memnew(LineEdit);
 			path_popup->add_child(path);
 			path->set_anchors_and_offsets_preset(PRESET_WIDE);
-			path->connect("text_entered", callable_mp(this, &AnimationTrackEdit::_path_entered));
+			path->connect("text_submitted", callable_mp(this, &AnimationTrackEdit::_path_entered));
```

Comment:
Missed `_path_entered` to `_path_submitted`

Reply:
Good catch! I should have paid more attention to my exclusion search filter :facepalm: 

Updated `expression.xml` and `animation_track_editor.cpp`. I've also squashed the commits into one, but I saved a branch locally that preserves the separate commits in case some one wants to review the commits separately